tests/services/test_feed_manager.py

```
@@ -0,0 +1,84 @@
+
+@pytest.fixture
+def feed_manager():
+    return FeedManager("feeds.opml")
+
+def test_load_valid_opml(feed_manager, valid_opml_content):
+    with patch("builtins.open", mock_open(read_data=valid_opml_content)):
+        valid_count, invalid_feeds = feed_manager.load_opml()
+        assert valid_count == 1
+        assert len(invalid_feeds) == 0
+
+def test_load_invalid_opml(feed_manager, invalid_opml_content):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_load_invalid_opml`

Add return type annotation: `None`

(ANN201)